src/main/db/defaults.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+const config = {
+  accounts: [],
+  counterValue: "USD",
+  language: null,
+  theme: "dusk",
+  region: null,
+  orderAccounts: "balance|desc",
+  countervalueFirst: true,
+  hasPassword: false,
+  autoLockTimeout: 10,
+  selectedTimeRange: "week",
+  marketIndicator: "western",
+  currenciesSettings: {},
+  pairExchanges: {},
+  developerMode: false,
+  loaded: true,
+  shareAnalytics: true,
+  sentryLogs: true,
+  lastUsedVersion: "1.18.2",
+  dismissedBanners: [],
+  accountsViewMode: "card",
+  showAccountsHelperBanner: true,
+  hideEmptyTokenAccounts: false,
+  sidebarCollapsed: false,
+};
```

Comment:
The defaults here are curious...
Why setting an arbitrary value to `lastUsedVersion` and `theme`, and `null` to `language` and `region`?

Reply:
this is a oopsie, those defaults aren't even used